hishel/_async/_transports.py

```diff
@@ -156,6 +156,7 @@ async def handle_async_request(self, request: Request) -> Response:
                     url=normalized_url(res.url),
                     headers=res.headers,
                     stream=AsyncCacheStream(res.stream),
+                    extensions=res.extensions,
```

Comment:
I thought we decided to use the original request timeout/extensions, for the revalidation request, from [this comment](https://github.com/karpetrosyan/hishel/issues/247#issuecomment-2190779262)

Reply:
Oh, seems like it's correct, we are taking the current request extensions, sorry

Reply:
Yes, after checking, as `construct_response_from_cache` will return the original request, I thought it was better to use this variable because if later this method changes something in there, it will get it as a bonus. And as all other params are constructed from this version of the request, lokked more logical to me.